routers/web/repo/view.go

```diff
@@ -927,12 +929,7 @@ func renderCode(ctx *context.Context) {
 		return
 	}
 
-	if !ctx.Repo.Repository.IsEmpty {
-		checkCitationFile(ctx, entry)
-		if ctx.Written() {
-			return
-		}
```

Comment:
Why was this removed?

Reply:
My mistake. Fixed in 9c503735a91e5f8d0bdf8b52a7a14978c5170b95